dts/bindings/charger/renesas,isl92xx.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2020 Intel Corporation
```

Comment:
this should come with the driver it supports, should be separate from the API introduction/discussion.